0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md

```diff
@@ -148,46 +168,51 @@ Content-Type: application/json
 
 ### Payment Setup
 
-The sender uses the receiver details to create the ILP packet:
+The sender uses the receiver details to create the PSK2 payment:
 
-* The `account` in the ILP packet is the `destination_account` provided by the receiver
-* The `amount` is determined by the sender:
+* The `destination_account` should be used as the PSK2 destinationAccount.
+* The `shared_secret` should be used as the PSK2 sharedSecret.
+* If present, the `balance.maximum` SHOULD be used as the PSK2 chunked payment amount.
 
-    * The sender uses the `ledger_info.currency_scale` to determine the integer amount to include in the ILP packet (e.g. if they want the receiver to receive 10 units of `ledger_info.currency_code` on their ledger, a `ledger_info.currency_scale` value of 2 would indicate they should set the `amount` to 1000). In cases where the sender knows the integer amount the receiver should get, for example if that amount was requested by the receiver out of band, the sender does not need to use `ledger_info.currency_code` and `ledger_info.currency_scale`
-    * The sender SHOULD NOT set an amount greater than the `maximum_destination_amount` or lower than the `minimum_destination_amount`, as this will be rejected by the receiver
+In a UI, the `ledger_info` and `receiver_info` objects (if present) can be used for display purposes. These objects can be manipulated by the receiver in any way, so amounts SHOULD be displayed in source units when possible.
 
-* The `data` is encoded using the [Pre-Shared Key protocol](../0016-pre-shared-key/0016-pre-shared-key.md):
+Note that the sender can send as many PSK2 payments as they want using the same receiver info. The sender SHOULD query the receiver again once the time indicated in the [`Cache-Control` header](#response-headers) has passed.
 
-    * The key is the `shared_secret` provided by the receiver
-    * Additional data may be included in the PSK details. SPSP data MUST be JSON.
-    * Private PSK headers SHOULD include `Content-Type: application/json` and `Content-Length: <byte length of data>`
+## Appendix A: Payment Pointer
 
-Note that the sender can send as many payments as they want using the same receiver info. The sender SHOULD query the receiver again once the time indicated in the [`Cache-Control` header](#response-headers) has passed.
+This is the recommended way to identify an SPSP receiver, and is intended to be the main form of identifier that users on Interledger will interact with. It can be used as a persistent identifier for a person or as a temporary identifier to represent an invoice, much like a bitcoin address.
 
-## Appendix A: (Optional) Webfinger Discovery
+The payment pointer is in the form `$example.com/bob` (A payment pointer with no path is also acceptable, i.e. `$bob.example.com`). This is converted into an SPSP URI, by removing the `$` and replacing it with `https://spsp.`.
```

Comment:
Would you read this as "pay bob dot example dot com"?

If you have your own domain, this format seems pretty good: "pay sharafian dot com".

I'm less sure about the experience for people that are addressed in terms of their provider. Email addresses read nicely as "evan at provider dot com". At seems a sensible way to indicate that it's a user with a certain provider (or maybe I'm just used to that now). "pay evan dot provider dot com" sounds a little less nice to me, maybe because I'm not really used to the dot separating the username from the provider. "pay provider dot com slash evan" is even worse, in my opinion.

Reply:
Email addresses have their quirks too; I think having our identifiers structured similar to web URLs makes them easier to understand. It also looks a little more like a modern handle-style identifier like `@example`, while `example@example.com` feels a little more dated (maybe just to me).

It's nice having the clear delineation between user and provider, but that also precludes the ability to have a short vanity address (ie. `$sharafian.com` instead of `pay@sharafian.com`) which I think is cool.

As for the path, I would imagine it would be used more commonly for things like invoices. Something like `$invoice.amazon.com/22589235` might read better than `$22589235.invoice.amazon.com`. It also helps us support people whose systems might not be easily set up to handle subdomains dynamically.

Reply:
>  I think having our identifiers structured similar to web URLs makes them easier to understand

This has not been true historically. The WebID standard has tried for a decade to get people to identify themselves using a URL with very little success.

> It also looks a little more like a modern handle-style identifier like @example

That's why Square used the CashHash format `$user`. The problem is that this is a domain-less identifier. You need to have a way to scope it. 

> example@example.com feels a little more dated (maybe just to me)

It's a compromise. If you've got the network effect of Twitter you can have an implicit domain but even then it doesn't always work: Does @sharafian mean https://twitter.com/sharafian or https://github.com/sharafian ?